realm/realm-library/src/main/cpp/io_realm_internal_SharedRealm.cpp

```diff
@@ -148,6 +148,24 @@ Java_io_realm_internal_SharedRealm_nativeGetVersion(JNIEnv *env, jclass, jlong s
     return -1;
 }
 
+JNIEXPORT void JNICALL
+Java_io_realm_internal_SharedRealm_nativeSetVersion(JNIEnv *env, jclass, jlong shared_realm_ptr, jlong version)
```

Comment:
Uh, the other one is `Java_io_realm_internal_SharedRealm_nativeGetVersionID()` ... Sorry... Maybe change them both to `SchemaVersion()`? But let's do it in another PR ... they are not related with this one.